pkg/api/v1/types.go

```diff
@@ -1352,6 +1352,16 @@ const (
 	PullIfNotPresent PullPolicy = "IfNotPresent"
 )
 
+// TerminationMessagePolicy describes how termination messages are retrieved from a container.
+type TerminationMessagePolicy string
+
+const (
+	// FallbackToLogsOnErrorTerminationMessage means that if the termination message file is empty, the most
+	// recent contents of the container logs will be used as the termination message if the container exits
+	// with an error.
+	FallbackToLogsOnErrorTerminationMessage TerminationMessagePolicy = "FallbackToLogsOnError"
```

Comment:
Hm, shouldn't this be `TerminationMessagePolicyReadLogsOnError` and I think you should add `TerminationMessagePolicyReadFilePath` or some such and make it the default. Then it would look intentional rather than something clearly tacked-on. :)